app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @project.name %></h1>
+<h3>Material: <%= @project.material %></h3>
+<h3>Challenge Theme: <%= @project.challenge.theme %></h3>
+
+<h4>Number of Contestants: <%= @project.contestants.count %></h4>
```

Comment:
Next time, be sure to encapsulate the logic with `count` in a model method. Even though it's a simple calculation, it's still an AR calculation that makes a query to the db, so it should be implemented in a model method. Then, you can call that model method from this @project instance. Also, when you add the model method, please test it!

Note: the rubric allows one MVC infraction so this doesn't affect your score